UGW: Add uClibc in libc file spec to use openwrt toolchain as external toolchain

diff --git a/package/libs/toolchain/Makefile b/package/libs/toolchain/Makefile
--- a/package/libs/toolchain/Makefile
+++ b/package/libs/toolchain/Makefile
@@ -163,8 +163,7 @@ define Package/libc/config
 		string
 		prompt "libc shared library files (use wildcards)"
 		depends on EXTERNAL_TOOLCHAIN && PACKAGE_libc
-		default "./lib/ld{-*.so,-linux*.so.*} ./lib/lib{anl,c,cidn,crypt,dl,m,nsl,nss_dns,nss_files,resolv,util}{-*.so,.so.*}"
-
+		default "./lib/ld{-*.so*,-linux*.so.*} ./lib/lib{anl,c,uClibc,cidn,crypt,dl,m,nsl,nss_dns,nss_files,resolv,util}{-*.so,.so.*}"
 	endmenu
 endef
 
